src/tests/dictionaryConcepts/container/EditConcept.test.jsx

```diff
@@ -252,17 +257,127 @@ describe('Test suite for mappings on existing concepts', () => {
     expect(instance.state.mappings.length).toEqual(1);
   });
 
-  it('should call removeMappingRow function', () => {
-    const event = { target: { tabIndex: 1 } };
+  it('should call removeConceptMappingAction function', () => {
```

Comment:
Are you testing removeMappingRow or removeConceptMappingAction?

Reply:
The function removeMappingRow has removeConceptMappingAction inside it that set the concent retired attribute to true. So am expecting that ```expect(instance.state.mappings[1].retired).toEqual(true)```. Although I have to call the removeMappingRow function for this test to work. what changes retired to true for this test to pass is the removeConceptMappingAction. So I am testing for removeConceptMappingAction

Reply:
I have ```it('should call removeMappingRow function``` on line 299